lambdas/db-migration/src/migrations/20201014104958_create_collections_table.ts

```diff
@@ -39,9 +39,6 @@ export const up = async (knex: Knex): Promise<void> =>
       .comment(
         'Duplicate handling behavior for this collection'
       );
-    table
```

Comment:
I just noticed this and wanted to give you a heads up that you can't remove columns from the table definitions in the existing migrations to get the desired effect. These migrations update the database schema and this particular file to create the collections table is already included in the 9.0.0 release that users will get. And once a schema migration has been run, it is not run again.

So if we want to remove a column/add a column/change a column in that table, then we have to add a **new** migration to this lambda to do that. The README in this lambda walks through the process of creating a new migration. And the knex docs on schema building are pretty good:

http://knexjs.org/#Schema-Building

Reply:
Ah gotcha. Thanks for the heads up! Will make some updates